client/app/pages/query-snippets/QuerySnippetsList.less

```
@@ -0,0 +1,9 @@
+.snippet-content {
+  max-width: 500px;
+  max-height: 56px;
+  overflow: hidden;
+  /* autoprefixer: off */
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+}
```

```suggestion
  white-space: pre-wrap;
}
```

Preserve line breaks - less messy, query preview looks more familiar to user.